src/python/dxpy/dx_extract_utils/filter_to_payload.py

```diff
@@ -119,9 +119,9 @@ def location_filter(location_list):
         # First, ensure that the geno bins width isn't greater than 250 megabases
         start = int(location["starting_position"])
         end = int(location["ending_position"])
-        if end - start > 250000000:
+        if end - start > 5000000:
```

Comment:
I think there is no e2e tests. Is it cover by unit test or do we want to add e2e test?